lib/key-info.js

```diff
@@ -0,0 +1,55 @@
+const xpath = require("xpath");
+
+/**
+ * A basic, string-based, implementation of a KeyInfoProvider
+ *
+ * @param {string} cert the public certificate
+ * @implements {KeyInfoProvider}
+ */
+class KeyInfo {
+  constructor(cert) {
+    this.cert = cert;
```

Comment:
Specify this should be base64 DER, and not PEM, or strip the tags from PEM.

Reply:
Do you think a comment to that effect would be sufficient at this time?